apps/experiments/versioning.py

```diff
@@ -78,8 +89,16 @@ def get_field(self, field_name: str) -> VersionField:
 
     @property
     def fields_grouped(self):
-        # TODO: Return fields with group info for display purposes
-        pass
+        groups = defaultdict(dict)
```

Comment:
It doesn't seem like this needs to be a default dict. Also they dict values are not `dict` so it also doesn't seem correct.

Reply:
Thanks. I was first going to use one, but later pivoted and didn't update this :+1: